resources/rafter/charts/controller-manager/charts/minio/templates/rafter-minio-envs.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "minio.fullname" . }}-envs
+data:
```

Comment:
Please add:
APP_INTERNAL_ENDPOINT: http://rafter-minio.kyma-system.svc.cluster.local:9000
and use it in rafter-upload-service deployment.